pkg/installer/installer.go

```diff
@@ -535,6 +539,7 @@ func checkAvailableDiskSpace(fsDisk disk, paths ...string) error {
 		if err != nil {
 			return err
 		}
+		span.SetTag(fmt.Sprintf("path.%s.available", path), int64(s.Available))
```

Comment:
I think it'd be better if you tagged the Span with the path, and set a `required_disk_space` tag instead

Reply:
there are multiple paths being checked